src/sentry/scripts/tsdb/cmsketch.lua

```diff
@@ -318,6 +318,166 @@ function Sketch:increment(items)
     return results
 end
 
+local function response_to_table(response)
+    local result = {}
+    for i = 1, #response, 2 do
+        result[response[i]] = response[i + 1]
+    end
+    return result
+end
+
+function Sketch:export()
+    -- If there's no data, there's nothing meaningful to export.
+    if not self:exists() then
+        return cmsgpack.pack(nil)
+    end
+    return cmsgpack.pack({
+        response_to_table(redis.call('ZRANGE', self.index, 0, -1, 'WITHSCORES')),
+        response_to_table(redis.call('HGETALL', self.estimates)),
+    })
+end
+
+function table.is_empty(t)
+    return next(t) == nil
+end
+
+function Sketch:import(payload)
+    local data = cmsgpack.unpack(payload)
+    if data == nil then
+        return  -- nothing to do here
+    end
+
+    local source_index, source_estimators = unpack(data)
+
+    -- TODO: This is absolutely not efficient when we're merging multiple things into the same sketch.
+    local destination_index = response_to_table(redis.call('ZRANGE', self.index, 0, -1, 'WITHSCORES'))
```

Comment:
We don't use `destination_index` in all cases, let's wait to do the zrange call until we know we need it?